AdvLoggerPkg/Include/Guid/AdvancedLoggerPreDxeLogs.h

```diff
@@ -0,0 +1,28 @@
+/** @file
+  This file defines GUIDs and data structure for producing HOB entry describing
+  the location of pre-DXE logs. For platforms which use a Pre-DXE AdvancedLoggerLib
+  implementation, this HOB is not necessary.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __ADVANCED_LOGGER_PRE_DXE_LOGS_H__
+#define __ADVANCED_LOGGER_PRE_DXE_LOGS_H__
+
+typedef struct _ADVANCED_LOGGER_PRE_DXE_LOGS_HOB {
+  UINT32    Signature;
+  UINT64    BaseAddress;
+  UINT64    LengthInBytes;

```

Comment:
Probably should specify what content are expected in this region, i.e. raw prints, or formatted adv logger entries.